### PR TITLE
add recipe for version 1.1.5 of biotransformer

### PR DESCRIPTION
**Pull request by martenson**
This is a new recipe for a software called Biotransformer. 

BioTransformer is a software tool that predicts small molecule metabolism in mammals, their gut microbiota, as well as the soil/aquatic microbiota. BioTransformer also assists scientists in metabolite identification, based on the metabolism prediction.

source: https://bitbucket.org/djoumbou/biotransformer
papers:
    - doi:10.1186/s13321‐018‐0324‐5 https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0324-5
    - doi:10.1093/nar/gkv1229 https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702869/

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/biotransformer/meta.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  description: |
+    BioTransformer is a software tool that predicts small molecule metabolism in mammals, their gut microbiota,
+    as well as the soil/aquatic microbiota. BioTransformer also assists scientists in metabolite identification,
+    based on the metabolism prediction.
+
+
+extra:
+  notes: |
+    Biotransformer is Java program that comes with a custom wrapper shell script.
+    This shell wrapper is called "opsin" and is on $PATH by default. By default
```
Comment (martenson) [inline recipes/biotransformer/meta.yml:43]:
```suggestion
    This python wrapper is called "biotransformer" and is on $PATH by default. By default
```

File: recipes/biotransformer/meta.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  summary: Predicts small molecule metabolism.
+  description: |
+    BioTransformer is a software tool that predicts small molecule metabolism in mammals, their gut microbiota,
+    as well as the soil/aquatic microbiota. BioTransformer also assists scientists in metabolite identification,
+    based on the metabolism prediction.
+
+
+extra:
+  notes: |
+    Biotransformer is Java program that comes with a custom wrapper shell script.
```
Comment (martenson) [inline recipes/biotransformer/meta.yml:42]:
```suggestion
    Biotransformer is Java program that comes with a custom wrapper python script.
```

**Comment by bgruening** [comment]
The jar file is not called snapshot imho:

grafik

**Comment by martenson** [comment]
@bgruening that is correct, thank you!

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Comment by bgruening** [comment]
Thanks @martenson!

**Comment by martenson** [comment]
Thanks for the review and merge @bgruening !

**Pull request merged**